docs/shop_administration/shop_administration.md

```diff
@@ -8,9 +8,8 @@ Depending on the chosen data provider and whether an ERP is connected, some tabs
 
 |Tab|Description|
 |--- |--- |
-|[Cockpit](cockpit.md)|Basic statistics|
+|[Dashboard](dashboard.md)|Basic statistics|
 |[Components](components.md)|A folder where translations are stored|
 |[Configuration Settings](https://doc.ibexa.co/en/latest/guide/shop_configuration/)|Basic shop configuration|
-|[Control center](control_center.md)|ERP information and management|
 |[Price and stock management](manage_prices_and_stock.md)|Product price and stock management|
 |[Order Management](manage_orders.md)|Basic order management|
```

Comment:
There is a screenshot with already removed items. It should be replaced or maybe deleted?

Reply:
The screenshot will be updated with the rest in user guide, once the new UI is fixed